Core/PixelEvent.swift

```diff
@@ -416,7 +415,7 @@ extension Pixel {
         case networkProtectionTunnelFailureDetected
         case networkProtectionTunnelFailureRecovered
 
-        case networkProtectionLatency(quality: NetworkProtectionLatencyMonitor.ConnectionQuality)
+        case networkProtectionLatency(quality: String)
```

Comment:
Changing just these allows us to remove NetworkProtection as dependency of Core and "flatten" the dependency tree, solving possible runtime issues during testing

Reply:
@diegoreymendez Do you have anything against this change? I can explain the rationale if needed.